recipes/sage-proteomics/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+rust_compiler_version:
+  - "1.79"
```

Comment:
I'm not in the rust team so this is more of a clarification question than a blocker. Why do you need to pin the compiler here? The rest seems good to merge.

Reply:
because this is what the upstream version required (compiled last year). It's not specified explicitly upstream but I found it after looking into the upstream CI logs.